offchainreporting/gethwrappers/compile.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+set -e
+
+solpath="$1"
+solcoptions=("--optimize" "--optimize-runs" "40000")
```

Comment:
To my knowledge, we always use 200 runs.  Any reason to bump it in this case?

Reply:
Runs is confusingly named. It's not the number of passes of the optimizer, but rather how many times you expect to execute ("run") the logic in the contract.

We do expect to run the contract thousands and thousands of times, so we want the optimizer to prioritize runtime cost over deployment cost/codesize.